src/main/java/org/folio/spring/filter/TenantOkapiHeaderValidationFilter.java

```diff
@@ -22,7 +22,8 @@
   @Override
   public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain) throws IOException, ServletException {
     HttpServletRequest req = (HttpServletRequest) request;
-    if (StringUtils.isBlank(req.getHeader(XOkapiHeaders.TENANT))) {
+    if (!req.getServletPath().startsWith("/admin/") // don't require x-okapi-tenant when calling admin endpoints
```

Comment:
I think will be good to have the ability to change this prefix if needed. I suggest creating a property with the default value `admin`.

Reply:
I believe the project has agreed upon /admin/health, but I will make it configurable

Reply:
Maybe we should provide a list of paths to exclude from the validation, instead of a prefix? 

We had similar case for swagger docs (/swagger-ui URI).

Reply:
> Maybe we should provide a list of paths to exclude from the validation, instead of a prefix?

I personally feel that's overkill here.  What's the use case?  Any requests coming from OKAPI should have the x-okapi-tenant header.  The intent of this exception/special case is to allow the API to be called outside a tenant context, by say a container management system.

Reply:
@bohdan-suprun I apologize - I missed that you DID provide a use case.